Sources/Core/AppEvents/AppEventsLogger.swift

```diff
@@ -141,6 +141,18 @@ public class AppEventsLogger {
   }
 
   //--------------------------------------
+  // MARK: - Push Notification
+  //--------------------------------------
+
+  /**
+   Sets a device token to register the current application installation for push notifications.
+   - parameter deviceToken: Device token data.
+  */
+  public static func setPushNotifications(deviceToken: Data!) {
```

Comment:
Let's convert this into a static var, like this:
```swift
public static var pushNotificaitonsDeviceToken: Data? {
    didSet {
      FBSDKAppEvents.setPushNotificationsDeviceToken(pushNotificaitonsDeviceToken)
    }
  }
```

Also, if you could split this into a separate PR, would help improve visibility.